### DISCUSSION TITLE
[Markdown] Probably an incorrect implementation of the strikethrough rule

### DISCUSSION THREAD

**Discussion by olegbarabanov** [community/community#18338] [Projects and Issues]
I noticed to the possible incorrect processing of the strikethrough rule in the Markdown editor. According to the Markdown rules, the text must be wrapped with **double tildes** for strikethrough (see [the section in GFM](https://github.github.com/gfm/#strikethrough-extension-)). But the Markdown editor does strikethrough even in the case of **single tildes**, which is not according to the specification, and can also lead to incorrect understanding of the information.

Correct example:
> "~~it must be strikethrough~~"

And in this example, single tildes should just wrap the text, *without strikethrough*:
> "~it must not be strikethrough~"  <= it must be like: \~it must not be strikethrough\~

Thanks.

**Comment by Nishant1500** [comment]
If i remember correctly, GitHub Flavoured Markdown have Flavoured in its name because they are some changes, a lot are changed to make it easy for users and unique in GitHub.

You should follow this guide instead
[**`Introduction to GitHub Flavoured Markdown`**](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

**Reply by olegbarabanov** [reply]
Thank you.

But even in the guide you mentioned ([Introduction to GitHub Flavoured Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)), it is also specified that to [strikethrough the text](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text), it is required to wrap with double tildes.

**Reply by Nishant1500** [reply]
Oh wow, i also realised. Probably a bug
